cape_encrypt/README.md

```diff
@@ -0,0 +1,22 @@
+Cape Encrypt
+===========
+
+Ease of use utility to allow functions deployed with [Cape](https://docs.capeprivacy.com/getting-started) to leverage enclave only encryption and decryption. These operations are completed using your account level [Cape Key](https://docs.capeprivacy.com/concepts/encrypt).
+
+
+## Installation 
+```sh
+pip install cape_encrypt
+```
+
+
+## Usage
+
+```python
+def cape_handler(data):
+    plaintext = b"encrypt me"
+    ciphertext = encrypt(plaintext)
+
+    print(str(ciphertext))
+    # cape:aaMrznPDy8ZrZT7CkRwpzTS9+rG+A4FuFPCfjLTrMa/2TyVvYdEx6PtYHnGizZKOZCytQUuo65OZkw5kQCJpNBuZzmd4lPB9lu0sSGLExPSivzMLKOH07rwrlBFFEEgZCvRoKBDebxVfq/Uv2v++Q4xmn4wksBjpPHjlLtGzpPu9mwMofs5eZLTVqp4g6yCuwaNbPkyhq09iRHiLOvWKhWfkf+0++/W2UDr81PLTdNBKI+kdHoTp/Xr8Uh9ooovwAx3V/LX9ESAHFWeW6BHV6JVcIP/tH1aFjuVVfH610I4eNZdaVyWV9DVdmsUF2o7g2tUmR+Eg++ts7MXxbWRz2PDZC8MDz52w6ZiUVluiluVPRh/VB+TmCJwSIfDQ3fiXAobhU/flA8jmzdE1pC3SdSY30vkqxwLBZ5VwGM4J7p2UsDuKzxZXVJ0Tg6ludB8y0NyZswYXZcewUuc0XZ2sOWCTqSP9t/0b/atGuwUxE5qkwEglP6s5AyVET8AZRH4KPoQuxjFUf7h+NJzZMDd/2Zef+yCGAP/8vKjpglDdItmsX3Bintu+Sp/ij6ynbFARpL9N7YZ8yA2Lpx/59Y/EnCuOdAJOpKcif3bnHNhKsGIATlO/lyY5bXRzGUpbejh+UAQC5qAsLmWQa/HZoF2ptGaGVhLpUs8zIdeLWFZ/YIhUXE1koI/BMMAT05kmaPPDvAELOkWJxpC4VYJWmzPZ29Opv7ye
```

Comment:
i'm not sure that this fits my mental model of the use case for cape_encrypt. i would think that decrypting inputs and then encrypting outputs would be the primary usage, e.g. 
```python
def cape_handler(data):
    plaintext = cape_encrypt.decrypt(data)
    result = do_something(plaintext)
    encrypted_result = cape_encrypt.encrypt(plaintext)
    return encrypted_result
```
but maybe i'm missing something in the weeds!

Reply:
Right now we're auto decrypting the inputs to cape_handler so until we turn that off it wouldn't quite make sense. I believe an example use case is from the original doc 

>they could create an asymmetric key-pair inside the enclave, and use cape.encrypt() on the private key before returning both to the function caller. Data could then be encrypted anywhere outside the enclave using the public key. The encrypted private key could be passed into a Cape function, and would automatically be decrypted, such that the function writer could decrypt their own data that was encrypted with their own public key.

Reply:
imo once this package is released we should prob turn off auto-decrypting, but ya that makes sense. maybe we can just improve on the usage examples in a future PR